core/storage/exploration/gae_models.py

```diff
@@ -333,6 +343,11 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single unshared instance."""
+        return base_models.EXPORT_METHOD.SHARED_INSTANCE
```

Comment:
Does not match docstring.

I'm going to stop here for the day -- PTAL and fix the docstrings throughout, then reassign back to me. Thanks!

Reply:
Done